test/main/store/actions/index.test.js

```diff
@@ -925,6 +926,55 @@ describe('#updateNetwork', () => {
       }
     })
   })
+
+  it('should correctly update the networksMeta', () => {
+    const icon = 'http://icon.com'
+    const nativeCurrencyIcon = 'http://icon2.com'
+    const nativeCurrencyName = 'TEST_NAME'
+    const symbol = 'TEST'
+    updateNetwork(
+      { id: '0x4', type: 'ethereum', name: '', explorer: '', symbol: '' },
+      {
+        id: '0x4',
+        type: 'ethereum',
+        name: 'test',
+        explorer: 'explorer.test',
+        symbol,
+        nativeCurrencyName,
+        nativeCurrencyIcon,
+        icon
+      }
+    )
+
+    expect(main.networksMeta.ethereum[4]).toStrictEqual({
+      icon,
+      nativeCurrency: { symbol, name: nativeCurrencyName, icon: nativeCurrencyIcon },
+      symbol
+    })
+  })
+
+  it('should not save network metadata inside the main network state', () => {
```

Comment:
I don't think we need this test, this isn't verifying any functionality only checking for static code correctness

Reply:
if anything maybe we can (or do) have separate tests that test updating the network somehow and those would fail if this wasn't right